### ISSUE TITLE
Script to deploy a node that joins an existing Private Tangle

### ISSUE THREAD

**Issue by jmcanterafonseca-iota** [iotaledger/one-click-tangle#10]
Once a new minimal Private Tangle is set up with the `private-tangle.sh` script (Coo + initial Node + Spammer) it would be good to have a script that allows to deploy extra Hornet Nodes which just join an existing Private Tangle and existing Node peers. Something like

```
private-tangle-node.sh --coo=<Coordinator Address> --peerWith=<Nodes to Peer with>
```

**Comment by islopma** [comment]
Hi @jmcanterafonseca-iota. Did you have any work done for this or any general idea on how to achieve this?

I'm very interested in adding some nodes to the Private Tangle. I don't have any clue now on how to do it but would love to contribute in any way I can.

(BTW, I love the Private Tangle.)

**Comment by jmcanterafonseca-iota** [comment]
@islopma that could be done, it is relatively easy, would you try?

The steps would be:

We can have an additional script under the `hornet-private-net` folder with the following interface:

```
node.sh [install|stop|start| <node_name>
````

For `install` the script creates a new folder, for instance `extra-nodes`( if not created yet), and under that folder a subfolder with the `node_name`. Additionally a `config` subfolder. Under the latter folder the `config-node.json` and `peering-node.json` from the private-tangle regular config are copied but renamed to `config.json` and `peering.json`. Also the docker-compose from `hornet-mainnet` is copied to the node folder (parent folder of `config`) so that now we have a docker-compose ready to instantiate a single node. The only tweak on that file is the name of the node, but that can be easily changed with a sed instruction. 

Afterwards we just need to generate an identity for our node, 

https://github.com/iotaledger/one-click-tangle/blob/chrysalis/hornet-private-net/private-tangle.sh#L268 

set it up

https://github.com/iotaledger/one-click-tangle/blob/chrysalis/hornet-private-net/private-tangle.sh#L268

and set up the Coo public key as here:

https://github.com/iotaledger/one-click-tangle/blob/chrysalis/hornet-private-net/private-tangle.sh#L222

finally `docker-compose -d up` should start our new Node connected to our Private Tangle.

**Comment by jmcanterafonseca-iota** [comment]
fixed by #43
